challenge.md

```diff
@@ -6,19 +6,34 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+> git clone {ssh url}
+
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+> git status
+
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
 
+> I dont know
+
+
 4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What you we do? What git command would one use?
 
+> You would create a new branch by doing this:
+> git checkout -b {name of branch}
+
 
 5. I want to add a file to the staging area. What git commit would I use?
 
+> git add
+
 
 6. I've been working on a branch, add-directories, to implement a feature. The feature is done and you need to add your changes into the master branch. What git command should I use?
 
+> git pull {ssh url to my repo} master
```

Comment:
Not really... We're talking about a single repo here, not sharing changes between repos. 

We've made a whole bunch of changes on the `add-directories` branch, and now we want to incorporate those changes into the `master` branch. To do this we first need to move to the `master` branch (`git checkout master`) and then run `git merge BRANCH-TO-MERGE-IN` (in this case, `add-directories`).